ui/panels-plugin/src/plugins/time-series-chart/time-series-chart-model.ts

```diff
@@ -137,7 +137,8 @@ export const PANEL_HEIGHT_LG_BREAKPOINT = 300;
 export const LEGEND_HEIGHT_SM = 40;
 export const LEGEND_HEIGHT_LG = 100;
 
-export const MIN_VALUE_PADDING_MULTIPLIER = 0.8;
+export const POSITIVE_MIN_VALUE_MULTIPLIER = 0.8;
+export const NEGATIVE_MIN_VALUE_MULTIPLIER = 1.2;
```

Comment:
Why do we use different multipliers for pos/neg values? It might be worth putting a comment explaining the reasoning for future us.

Reply:
Good call